File: AdminAPI/Models/Activity.cs
namespace AdminAPI.Models;

public class Activity
{
    public int Id { get; set; }
    public string Title { get; set; } = string.Empty;
    public string? Description { get; set; }
    public string? IconClass { get; set; }
    public int SortOrder { get; set; }
    public DateTime CreatedAt { get; set; }
    public string? ImageUrl { get; set; }
}

File: AdminPanelUI/src/types/activity.ts
export interface Activity {
  id: number
  title: string
  description: string | null
  sortOrder: number
  imageUrl: string | null
  createdAt: string
}

export interface SaveActivityPayload {
  title: string
  description: string | null
  sortOrder: number
  imageFile?: File
}
